### PR TITLE
[Merged by Bors] - Bump crate versions and VERSION for 0.7.4

### PR DESCRIPTION
**Pull request by nicholastmosher**

### CODE REVIEW

File: src/auth/Cargo.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ tracing = "0.1.21"
 tracing-futures = "0.2.4"
 x509-parser = "0.9.1"
 
-fluvio-controlplane-metadata = { version = "0.7.0", path = "../controlplane-metadata" }
+fluvio-controlplane-metadata = { version = "0.8.0", path = "../controlplane-metadata" }
```
Comment (sehz) [inline src/auth/Cargo.toml:25]: reason for updating to `0.7.0`?

**Review by sehz — CHANGES_REQUESTED** [review]

File: src/auth/Cargo.toml (hunk 1 above)
Comment (nicholastmosher) [reply src/auth/Cargo.toml:25]: Yes, we changed `TopicSpec::partitions() -> Option<i32>` to `TopicSpec::partitions() -> i32` which is a breaking change

Comment (sehz) [reply src/auth/Cargo.toml:25]: Got it

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**